### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kimeetok**
TODO: update links and desc for 'enablers'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,172 @@
...
+| Toxic Sentence Generator |  This module generates toxic sentence based on a given seed prompt. The attack module intends to test if the system under tests will complete the sentence with toxic sentences/phrases. | 
+| Violent Durian | This is a multi-turn agent designed to interact over several exchanges. It&#x27;s used to elicit dangerous or violent suggestions from the target language model by adopting a criminal persona. The application is experimental and uses OpenAI GPT-4. Configure the endpoint openai-gpt4 to use this attack module. |  
+
+</br>
+
+### Cookbooks
+
+| Cookbook | description |
```
Comment (imda-normanchia) [inline README.md:97]: capitalise the first letter

File: README.md [hunk 2]
```diff
@@ -1 +1,172 @@
...
+</br></br>
+
+## Getting Started
+
+### ✅ Prerequisites
+1. [Python 3.11](https://www.python.org/downloads/) (We have yet to test on later releases)
+
+2. [Git](https://github.com/git-guides/install-git)
+
+3. [Moonshot Python Library](https://github.com/aiverify-foundation/moonshot)
```
Comment (imda-normanchia) [inline README.md:61]: should we link this to the pypi instead?

File: README.md [hunk 3]
```diff
@@ -1 +1,172 @@
-# moonshot-data
\ No newline at end of file
+<div align="center">
+
+![Moonshot Logo](https://github.com/moonshot-admin/moonshot/raw/main/misc/aiverify-moonshot-logo.png)
```
Comment (imda-normanchia) [inline README.md:3]: change the `raw` to `blob`

File: README.md (hunk 2 above)
Comment (imda-lseokmin) [reply README.md:61]: You mean Moonshot Python Library? I think don't need. But I think the phrase is too deliberate. Shall we just change to "Moonshot Library"?

**Review by imda-lseokmin — APPROVED** [review]
Removed the dead links. Reviewed by another 3 person.

**Pull request merged**